drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -6,7 +6,7 @@
 
 #define DT_DRV_COMPAT espressif_esp32_wifi
 
-#define _POSIX_C_SOURCE 200809
+#define _POSIX_C_SOURCE 200809L
```

Comment:
This can be removed as there is no non-standard posix call in Espressif's sources.

Reply:
Thanks! I'll mix that in once I've got things building.

Reply:
Looks like that worked!